<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartingWindow</class>
 <widget class="QMainWindow" name="StartingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>velas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Inicial"/>
      <widget class="QWidget" name="newRecurso">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecursos">
    <property name="title">
     <string>Recursos</string>
    </property>
    <addaction name="actionCrear_recurso"/>
    <addaction name="actionEliminar_recurso"/>
    <addaction name="actionModificar_recurso"/>
   </widget>
   <widget class="QMenu" name="menuProductos">
    <property name="title">
     <string>Productos</string>
    </property>
    <addaction name="actionCrear_producto"/>
    <addaction name="actionEliminar_producto"/>
   </widget>
   <addaction name="menuRecursos"/>
   <addaction name="menuProductos"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCrear_recurso">
   <property name="text">
    <string>Crear recurso</string>
   </property>
  </action>
  <action name="actionEliminar_recurso">
   <property name="text">
    <string>Eliminar recurso</string>
   </property>
  </action>
  <action name="actionModificar_recurso">
   <property name="text">
    <string>Modificar recurso</string>
   </property>
  </action>
  <action name="actionCrear_producto">
   <property name="text">
    <string>Crear producto</string>
   </property>
  </action>
  <action name="actionEliminar_producto">
   <property name="text">
    <string>Eliminar producto</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
